test/live_queue_test.exs

```diff
@@ -67,8 +67,9 @@ defmodule Membrane.VideoCompositor.LiveQueueTest do
   test "if sends stream format and scene only once" do
     state = setup_videos() |> send_both_pads_frames()
 
-    {_actions, state} = LiveQueue.handle_tick(:buffer_scheduler, %{}, state)
-    {actions, _state} = LiveQueue.handle_tick(:buffer_scheduler, %{}, state)
+    {actions_1, state} = LiveQueue.handle_tick(:buffer_scheduler, %{}, state)
+    {actions_2, _state} = LiveQueue.handle_tick(:buffer_scheduler, %{}, state)
+    {actions_3, _state} = LiveQueue.handle_tick(:buffer_scheduler, %{}, state)
```

Comment:
Shouldn't you pass the state from step 2. to the third `handle_tick` invocation?

Reply:
Fixed